yas3fs/__init__.py

```diff
@@ -1211,7 +1211,13 @@ def delete_cache(self, path):
 
     def process_message(self, messages):
         logger.debug("process_message '%s'" % (messages))
-        c = json.loads(messages)
+        # Discard malformed JSON https://github.com/danilop/yas3fs/issues/141
+        try:
+            c = json.loads(messages)
+        except Exception as e:
+            logger.debug("process_message discarding malformed message")
```

Comment:
Should this be more severe? (Warning, maybe even error?)
Also, is Exception too broad? (Probably harmless here, though.)